### DISCUSSION TITLE
Is that possible to override the lazygit keymap in plugin?

### DISCUSSION THREAD

**Discussion by linsui** [LazyVim/LazyVim#488] [Q&A] [answered]
I don't use lazygit. I want to use the <leader>gg keymap for neogit. But it seems adding the keymap in neogit's config is not enough. I have to set it in keymap.lua. Is it possible to do that in plugin only?

**Comment by folke** [comment]
You can set it for neogit using the lazy,nvim `keys=` property

**Reply by linsui** [reply]
I tried that but it doesn't work. I have to set it in keymap.lua.

**Reply by linsui** [reply]
```
    "TimUntersberger/neogit",
    cmd = "Neogit",
    keys = { { "<leader>gg", "<cmd>Neogit<CR>", mode = { "n" }, desc = "Open Neogit" } },
```

This is my neogit config.

**Reply by folke** [reply]
There's indeed a bug in lazy.nvim. I'll fix it, but for now, just set mode="n", That will work

**Reply by linsui** [reply]
Thanks!

**Reply by folke** [reply, marked as answer]
Just fixed it in lazy.nvim. Will create a new release in a bit
